src/nav/MainNavBar.js

```diff
@@ -22,30 +22,34 @@ type Props = {
   backgroundColor: string,
   unreadMentionsTotal: number,
   onPressStreams: () => void,
+  orientation: string,
 };
 
 class MainNavBar extends PureComponent<Props> {
   static contextTypes = {
     styles: () => null,
+    navBarSizes: () => null,
   };
 
   props: Props;
 
   render() {
-    const { styles } = this.context;
+    const { styles, navBarSizes } = this.context;
     const {
       actions,
       backgroundColor,
       textColor,
       unreadMentionsTotal,
       onPressStreams,
       editMessage,
+      orientation,
     } = this.props;
 
     const leftPress = editMessage ? actions.cancelEditMessage : onPressStreams;
+    const navBarSize = orientation === 'PORTRAIT' ? navBarSizes.portrait : navBarSizes.landscape;
 
     return (
-      <View style={[styles.navBar, { backgroundColor }]}>
+      <View style={[styles.navBar, { backgroundColor }, { height: navBarSize }]}>
```

Comment:
You can have a navBarLandscape style that has the height set.

Reply:
@borisyankov OK, I created the `navBarLandscape` style in 0d86fe4. Thanks!